src/table/table.scss

```diff
@@ -4,383 +4,423 @@
 @import '../icon/index';
 @import './variables';
 
-@mixin iui-tables-table {
+@mixin iui-table {
   @include iui-reset;
-
   @include iui-font-family;
   display: flex;
   flex-direction: column;
-  padding: 0;
 
   * {
     box-sizing: border-box;
   }
 
-  > .iui-tables-body {
-    width: 100%;
-    @include themed {
-      background-color: t(iui-color-background-1);
-    }
+  .iui-table-header {
+    @include iui-table-header;
   }
 
-  &.iui-zebra-striping > .iui-tables-body > .iui-tables-row:nth-child(even) {
+  .iui-table-body {
+    @include iui-table-body;
+  }
+
+  .iui-cell {
+    @include iui-table-cell;
+  }
+
+  &.iui-zebra-striping > .iui-table-body > .iui-row:nth-child(even):not(.iui-selected) {
     @include themed {
       background: rgba(t(iui-color-foreground-body-rgb), 0.02);
     }
   }
 
-  &.iui-tables-fixed > .iui-tables-body {
-    max-height: 300px;
+  &.iui-fixed > .iui-table-body {
+    max-height: $iui-baseline * 27;
     overflow-x: hidden;
     overflow-y: scroll;
     overflow: overlay;
   }
+}
 
-  .iui-select .iui-select-button {
-    background-color: transparent;
-    border: none;
+@mixin iui-table-header { 
+  user-select: none;
+  @include themed {
+    background-color: t(iui-color-background-3);
   }
 
-  .iui-tables-row {
-    margin-top: -1px;
+  .iui-row {
     display: flex;
     flex-grow: 1;
-    width: 100%;
-    border-top: 1px solid transparent;
-    border-left: 1px solid transparent;
-    border-right: 1px solid transparent;
-
-    @include themed {
-      border-bottom: solid 1px t(iui-color-background-4);
-    }
+  }
 
-    > .iui-tables-slot > .iui-tables-menu {
-      visibility: hidden;
-    }
+  .iui-cell:not(.iui-slot) {
+    @include iui-focus;
+    cursor: pointer;
+    position: relative;
+    overflow: visible;
 
-    &:hover:not(.iui-tables-row-active, .iui-tables-paginator) {
-      @include themed {
-        background-color: rgba(
-          t(iui-color-foreground-primary-rgb),
-          t(iui-opacity-6)
-        ) !important;
-      }
+    > .iui-borderless {
+      margin-left: $iui-xs;
 
-      > .iui-tables-slot > .iui-tables-menu {
-        visibility: visible;
+      &:not(.iui-active) {
+        visibility: hidden;
       }
     }
 
-    > .iui-tables-cell {
-      align-items: center;
-      display: flex;
-      flex-grow: 1;
-      min-width: 145px;
-      padding: $iui-table-density;
-      overflow: hidden;
-      list-style: none;
-      flex-basis: 145px;
-
-      > svg {
-        width: $iui-icons-default;
-        height: $iui-icons-default;
-        margin-right: $iui-sm;
+    > .iui-resizer {
+      height: 100%;
+      width: $iui-m;
+      position: absolute;
+      top: 0;
+      right: 0;
+      transform: translateX(50%);
+      touch-action: none;
+      cursor: ew-resize;
+      z-index: 100;
```

Comment:
Needed? if yes, why 100 value...?

Reply:
We do need to apply a z-index, but am unsure what value is necessary.

With z-index, you can move the cursor slightly over the column to the right without the resizer disappearing.
![zindex](https://user-images.githubusercontent.com/849817/121257528-ccf3e280-c87b-11eb-9172-1f5f9b289ae6.gif)

When z-index is removed the right column steals the focus almost immediately.
![zindex-nope](https://user-images.githubusercontent.com/849817/121257623-e6952a00-c87b-11eb-8ebb-274e75772f45.gif)

Reply:
z-index should be the lowest value possible. So try with 1, 2, etc. I think that should be enough

Reply:
Im leaving now what it is and we need to create an item to investigate it.